src/main/java/com/cb/services/DBService/DBServiceImpl/UserDBServiceImpl.java

```diff
@@ -53,10 +53,30 @@ public int insertUser(UserDAL u) {
     public List<UserDAL> getUserByEmail(String email) {
 
         return template.query("SELECT * FROM users where email = '" + email + "'", new RowMapper<UserDAL>() {
+            @Override
+            public UserDAL mapRow(ResultSet rs, int rowNum) throws SQLException {
+                try{
+                    UserDAL u = new UserDAL();
+                    u.setEmail(rs.getString(4));
```

Comment:
DAL is the mirror of DB. You have to map concrete values from DAL to BL in the service.  Controller have to receive DTO with BL included.